### ISSUE TITLE
Displaying the application version does not work

### ISSUE THREAD

**Issue by mmiglier** [CouscousPHP/Couscous#127]
The --version option gives `Console Tool` as output instead of the current version:
```
$ couscous -V
Console Tool
```

**Comment by mnapoli** [comment]
True, for now we don't handle the version in the code because it would require to keep it up to date before tagging and making new releases. If we had some kind of release generation script, we could add it in there but that's not the case today. If we had to do it manually I know I would forget it half the time, so IMO it's better not to have a version number (for now) than have a wrong one :)

Let's keep the issue open to fix it in the future.

**Comment by wysow** [comment]
@mnapoli what do you think about doing this in the compile script maybe? Using a version.txt as container for the current version number maybe?

**Comment by mnapoli** [comment]
@wysow that would be great but we would have to keep it up to date, and I know I will forget about it :p

**Comment by wysow** [comment]
@mnapoli I'm wondering if this could be linked to #149 and maybe use Github releases to have a more robust solution...

**Comment by mnapoli** [comment]
Yes that would be great, but keep in mind that version numbers might not always follow in the correct order (e.g. if there's a bugfix in an older maintained version), so taking the latest release (by date) isn't always the "highest" release.
